<launch>

    <arg name="lidar_frame" default="rslidar" />
    <arg name="pointcloud_topic" default="rslidar_points" />

    <node pkg="laser_detector" type="laser_detector_node" name="laser_detector_node" output="screen">
        <param name="bound_left" value="-0.2" type="double"/>
        <param name="bound_right" value="0.2" type="double"/>
        <param name="bound_back" value="0" type="double"/>
        <param name="bound_front" value="0.5" type="double"/>
        <param name="bound_top" value="1.0" type="double"/>
        <param name="bound_bottom" value="-1.0" type="double"/>
        <param name="nums_of_obstacle" value="5" type="int"/>
        <param name="scan_frame" value="$(arg lidar_frame)" type="string"/>
        <!-- <param name="scan_topic" value="$(arg htc_lidar_scan_topic)" type="string"/> -->
        <param name="pointcloud_topic" value="$(arg pointcloud_topic)" type="string" />
        <param name="debug" value="true" type="bool"/>
        <param name="direction" value="0" type="int"/> 
    </node> 


</launch>
